alpha/apps/kaltura/lib/kObjectDeleteHandler.php

```diff
@@ -138,6 +138,15 @@ protected function entryDeleted(entry $entry)
 		}
 		
 		kJobsManager::addDeleteJob($entry->getPartnerId(), DeleteObjectType::CATEGORY_ENTRY, $filter);
+
```

Comment:
Maybe move this code to the peer and create a function like deleteEntryServerNodes and provide the entry id $entry->getId()